backend/src/main/java/org/example/backend/controller/ShopController.java

```diff
@@ -39,5 +38,9 @@ public ResponseEntity<ShoppingList> getListById(@PathVariable String id) {
         Optional<ShoppingList> list = shopService.getListById(id);
         return list.map(ResponseEntity::ok).orElseGet(() -> ResponseEntity.notFound().build());
     }
+    @GetMapping("/stats/{id}")
+    public String getToDoListenWithStats(@PathVariable String id) {
+        return shopService.getToDoListenWithStats(id);
```

Comment:
Bleibe konstant in der Sprache "Listen"

Reply:
wurde angepasst auf den englischen Begriff getToDoListsWithStats